pimd/pim_cmd.c

```diff
@@ -6315,41 +6351,74 @@ static void show_mroute_count(struct pim_instance *pim, struct vty *vty)
 		pim_inet4_dump("<source?>", sr->c_oil.oil.mfcc_origin,
 			       source_str, sizeof(source_str));
 
-		vty_out(vty, "%-15s %-15s %-8llu %-7ld %-10ld %-7ld\n",
-			source_str, group_str, sr->c_oil.cc.lastused,
-			sr->c_oil.cc.pktcnt - sr->c_oil.cc.origpktcnt,
-			sr->c_oil.cc.bytecnt - sr->c_oil.cc.origbytecnt,
-			sr->c_oil.cc.wrong_if - sr->c_oil.cc.origwrong_if);
+		if (uj) {
+
+			json_object_object_get_ex(json, group_str, &json_group);
+
+			if (!json_group) {
+				json_group = json_object_new_object();
+				json_object_object_add(json, group_str,
+						       json_group);
+			}
+
+			json_source = json_object_new_object();
+			json_object_object_add(json_group, source_str,
+					       json_source);
+
+			json_object_int_add(json_source, "lastUsed",
+					    sr->c_oil.cc.lastused);
+			json_object_int_add(json_source, "packets",
+					    sr->c_oil.cc.pktcnt);
+			json_object_int_add(json_source, "bytes",
+					    sr->c_oil.cc.bytecnt);
+			json_object_int_add(json_source, "wrongIf",
+					    sr->c_oil.cc.wrong_if);
+		} else
```

Comment:
Nit, but the style guide says that if one block is wrapped in braces, so should the other one be.

Reply:
Done , Added braces for else block as well.